modules/oe_translation_poetry/tests/Functional/PoetryConfigurationTest.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_translation_poetry\Functional;
+
+use Drupal\Component\Render\FormattableMarkup;
+use Drupal\Core\Url;
+use Drupal\oe_translation_poetry_mock\PoetryMock;
+
+/**
+ * Tests the configuration of Poetry translators works as intended.
+ */
+class PoetryConfigurationTest extends PoetryTranslationTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    /** @var \Drupal\user\RoleInterface $role */
+    $role = $this->entityTypeManager->getStorage('user_role')->load('translator');
```

Comment:
Why are you making the translator a tmgmt admin? If you need to edit the config, make a user for that, log in with that and then log in as translator.